FIXTURES.md

```diff
@@ -59,3 +59,4 @@ id|description|valid v1|valid v2
 060|A layer named "water" with a single feature and unique properties (used as a layer compositing test with fixture 059) - [spec source](https://github.com/mapbox/vector-tile-spec/blob/master/2.1/README.md)|:white_check_mark:|:white_check_mark:
 061|A linestring feature that includes a ClosePath command.  A tile with such a command was added as a test fixture in https://github.com/mapbox/mapbox-gl-native/pull/2721, suggesting that such tiles may exist in practice, e.g. in old mapbox-gl-native offline databases. - [spec source](https://github.com/mapbox/vector-tile-spec/blob/master/1.0.1/vector_tile.proto) - recommended error handling `fatal`|:white_check_mark:|:x:
 062|points with different values for the same property - helpful for filtering tests - [spec source](https://github.com/mapbox/vector-tile-spec/blob/master/2.1/README.md)|:white_check_mark:|:white_check_mark:
+063|Point features with all properties suffixed with "xx_". Helpful for testing search/dropping of properties based on a particular syntax. Also includes one value with a different suffix in the case where only some data is intended to be read/dropped. - [spec source](https://github.com/mapbox/vector-tile-spec/blob/master/2.1/README.md)|:white_check_mark:|:white_check_mark:
```

Comment:
This mentions prefix `xx_` but tests use `_mbx_name_`.

Also, make suggestions for "prefix" instead of "suffix".

```suggestion
063|Point features with all properties prefixed with "xx_". Helpful for testing search/dropping of properties based on a particular prefix. Also includes one value with a different prefix in the case where only some data is intended to be read/dropped. - [spec source](https://github.com/mapbox/vector-tile-spec/blob/master/2.1/README.md)|:white_check_mark:|:white_check_mark:
```